### ISSUE TITLE
Helm Release Github action doesn't work quite some time

### ISSUE THREAD

**Issue by drehelis** [grafana/oncall#3376]
### What went wrong?

It's seems like the [Helm Release](https://github.com/grafana/oncall/actions/workflows/helm_release.yml) is not working for quite some time now and there are no helm releases past since `1.3.45` Chart version.

### How do we reproduce it?

```
helm repo update

...Successfully got an update from the "grafana" chart repository
```

```
helm search repo grafana/oncall

NAME          	CHART VERSION	APP VERSION	DESCRIPTION
grafana/oncall	1.3.45       	v1.3.45    	Developer-friendly incident response with brill...
```

### Grafana OnCall Version

v1.3.59

### Product Area

Helm

### Grafana OnCall Platform?

None

**Comment by drehelis** [comment]
@joeyorlando Could you take a look on this please?

**Comment by joeyorlando** [comment]
hi there, thanks for the issue 👋 

It looks like there was a GitHub Action in that CI pipeline that was not listed as an "approved action" in our repo settings. I've since added it. The next release that we do _should_ properly cut the helm chart update.

**Comment by via-justa** [comment]
Can someone release a patch version that will update the chart?

**Comment by nniehoff** [comment]
see also #3194

**Comment by joeyorlando** [comment]
Closing as fixed starting with v1.3.60
